projects/driver_display/inc/driver_display_brightness.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+// Module used to control the brightness of the driver display screens.
+// Also defines display screen parameters such as frequency and number of screens
+
+#include "adc.h"
+#include "gpio.h"
+#include "pwm.h"
+#include "soft_timer.h"
+
+#define DRIVER_DISPLAY_CONFIG_NUM_SCREENS 2
+
+#define DRIVER_DISPLAY_CONFIG_SCREEN1_FREQ_HZ 30000  // Frequency in Hz (subject to change)
+#define DRIVER_DISPLAY_CONFIG_SCREEN2_FREQ_HZ 30000  // Frequency in Hz (subject to change)
+
+#define DRIVER_DISPLAY_CONFIG_REFRESH_PERIOD 5
+
+typedef enum {
+  DRIVER_DISPLAY_SCREEN1 = 0,
```

Comment:
This doesn't follow the naming convention. It should be prefixed with the module name.